zeek/README.md

```diff
@@ -0,0 +1,201 @@
+## Overview
+
+[Zeek][6] is the world's leading platform for network security monitoring. It interprets what it sees and creates compact, high-fidelity transaction logs, and file content. It can create fully customized output, suitable for manual review on disk or in a more analyst-friendly tool like a security and information event management (SIEM) system.
```

Comment:
Could we call this [Corelight / Zeek][6] instead of just Zeek?

Reply:
Since we call the integration Zeek and plan to support both Open Source Zeek and Corelight Zeek Implementation, Zeek seems reasonable but please let me know your views and we can discuss and make changes as needed.